catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/controller/NotificationController.java

```diff
@@ -193,15 +193,6 @@ public void deletePersistentNotification(
       throw new IllegalArgumentException("ServerMessage is null");
     }
 
-    Subject subject = null;
```

Comment:
❓ In the other methods, we seemed to be doing this to check if the user id was defined and threw an IllegalArgumentException if it wasn't. Granted the other method were using the user id after that and here we are not but isn't the check something we would want here too?